src/main/java/Duke.java

```diff
@@ -1,10 +1,221 @@
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
 public class Duke {
+    private static final String HORIZONTAL_LINE = "____________________________________________________________\n";
+    private static final String LOGO = " ______    _   __   __            \n" +
+            "|_   _ \\  (_) [  | [  |           \n" +
+            "  | |_) | __   | |  | |   _   __  \n" +
+            "  |  __'.[  |  | |  | |  [ \\ [  ] \n" +
+            " _| |__) || |  | |  | |   \\ '/ /  \n" +
+            "|_______/[___][___][___][\\_:  /   \n" +
+            "                         \\__.'    \n";
+    private static final String GREETING_MESSAGE = HORIZONTAL_LINE +
+            " Hello! I'm Billy\n" +
+            " What can I do for you?\n" +
+            HORIZONTAL_LINE;
+    private static final String GOODBYE_MESSAGE = HORIZONTAL_LINE +
+            " Bye. Hope to see you again soon!\n" +
+            HORIZONTAL_LINE;
+    private static final String HELP_MESSAGE = "Here's some tips on how to use me!\n\n" +
+            "todo [input]\n" +
+            "\t - Add a todo task to the list\n\n" +
+            "deadline [input] /by [input]\n" +
+            "\t - Add a deadline task by the given deadline\n\n" +
+            "event [input] /at [input]\n" +
+            "\t - Add an event task at the given time\n\n" +
+            "list\n" +
+            "\t - List out all the current tasks\n\n" +
+            "done [task_numbers]\n" +
+            "\t - Marks the given tasks as done\n\n" +
+            "bye\n" +
+            "\t - Terminates me :(\n";
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        // Initialize variables for program startup
+        System.out.println("Hello from\n" + LOGO + GREETING_MESSAGE);
+        boolean isProgramRunning = true;
+        String userInput;
+        List<Task> list = new ArrayList<>();
+        Scanner in = new Scanner(System.in);
+
+        while (isProgramRunning) {
+            userInput = in.nextLine().trim();
+            String userInputLowerCase = userInput.toLowerCase();
+
+            if (userInput.equalsIgnoreCase("bye")) {
+                System.out.println(GOODBYE_MESSAGE);
+                isProgramRunning = false;
+            } else if (userInput.equalsIgnoreCase("list")) {
```

Comment:
maybe can extract and refactor the command names and make them as constants
"done", "todo", etc